### PR TITLE
Added DNS stuff from Jan-Piet Mens (blogged info and a free book)

### PR DESCRIPTION
**Pull request by pcarboni**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,7 +224,15 @@ <h2><a name="documents">Reading material</a></h2>
       </tr>
       <tr>
         <td><a target="_blank" href="http://www.iana.org/domains">IANA Domain Name Services</a></td>
-        <td>IANA maintaines and operates several key aspects of the DNS</td>
+        <td>IANA maintains and operates several key aspects of the DNS</td>
+      </tr>
+      <tr>
+        <td><a target="_blank" href="https://jpmens.net/dns">Blogged DNS links listinformation listing</a></td>
```
Comment (Habbie) [inline index.html]: 'Blogged DNS links listinformation listing' looks weird, did you mean to write it like that?

File: index.html [hunk 2]
```diff
@@ -224,7 +224,15 @@ <h2><a name="documents">Reading material</a></h2>
...
+        <td>Compilation of blogged DNS links from Jan-Piet Mens</td>
+      </tr>
+      <tr>
+        <td><a target="_blank" href="http://mens.de/:/altdnsbook">Alternative DNS Servers (Free)</a></td>
+        <td>Book written by Jan-Piet Mens (link published with permission)</td>
```
Comment (Habbie) [inline index.html]: There's no need to say " (link published with permission)" :)

File: index.html (hunk 1 above)
Comment (pcarboni) [reply index.html]: Fixed! New text -> "Blogged DNS links listing"

File: index.html (hunk 2 above)
Comment (pcarboni) [reply index.html]: Ok. Fixed! (Removed)

**Pull request merged**